src/core/toga/app.py

```diff
@@ -545,6 +565,23 @@ def on_exit(self, handler):
     def add_background_task(self, handler):
         self._impl.add_background_task(handler)
 
+    def onSaveInstanceState(self, outState):
+        """Override this method in your Android toga.App to save the data of your
```

Comment:
So - this method is on the public interface. The fact that the method name is an exact copy of the Android name, and you reference Android in the docstring should be a red flag.